packages/rancher-istio-1.9/charts/app-readme.md

```diff
@@ -0,0 +1,45 @@
+# Rancher Istio
```

Comment:
We'll have to keep an eye on anything duplicated across the charts, to make sure they both get updated.